platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -161,6 +172,8 @@ def initialize_chassis_thermals():
                 count = DeviceDataManager.get_gearbox_count('/run/hw-management/config')
             elif 'CPU Core' in rule['name']:
                 count = DeviceDataManager.get_cpu_thermal_count()
+            elif 'SODIMM' in rule['name']:
```

Comment:
I didn't see the code fetching 'PCH temp'?

Reply:
right, cause 'PCH temp' is not with type 'indexable'...